src/modm/platform/clock/stm32/module.lb

```diff
@@ -57,6 +58,7 @@ def build(env):
     properties["pllprediv"] = \
         (target["family"] in ["f0", "f3"] or (target["family"] == "f1" and target["name"] in ["00", "05", "07"]))
     properties["pllprediv2"] = False    # FIXME: not sure what value this should have
+    properties["pll_hse_prediv2"] = target["family"] == "f1" and target["name"] in ["01", "02", "03"]
```

Comment:
The CMSIS header suggest there are more devices with this predivider?
```
 $ ack PLLXTPRE -l
stm32f0xx/Include/stm32f030x6.h
stm32f0xx/Include/stm32f030x8.h
stm32f0xx/Include/stm32f030xc.h
stm32f0xx/Include/stm32f031x6.h
stm32f0xx/Include/stm32f038xx.h
stm32f0xx/Include/stm32f042x6.h
stm32f0xx/Include/stm32f048xx.h
stm32f0xx/Include/stm32f051x8.h
stm32f0xx/Include/stm32f058xx.h
stm32f0xx/Include/stm32f070x6.h
stm32f0xx/Include/stm32f070xb.h
stm32f0xx/Include/stm32f071xb.h
stm32f0xx/Include/stm32f072xb.h
stm32f0xx/Include/stm32f078xx.h
stm32f0xx/Include/stm32f091xc.h
stm32f0xx/Include/stm32f098xx.h

stm32f1xx/Include/stm32f100xb.h
stm32f1xx/Include/stm32f100xe.h
stm32f1xx/Include/stm32f101x6.h
stm32f1xx/Include/stm32f101xb.h
stm32f1xx/Include/stm32f101xe.h
stm32f1xx/Include/stm32f101xg.h
stm32f1xx/Include/stm32f102x6.h
stm32f1xx/Include/stm32f102xb.h
stm32f1xx/Include/stm32f103x6.h
stm32f1xx/Include/stm32f103xb.h
stm32f1xx/Include/stm32f103xe.h
stm32f1xx/Include/stm32f103xg.h
stm32f1xx/Include/stm32f105xc.h
stm32f1xx/Include/stm32f107xc.h

stm32f3xx/Include/stm32f301x8.h
stm32f3xx/Include/stm32f302x8.h
stm32f3xx/Include/stm32f302xc.h
stm32f3xx/Include/stm32f302xe.h
stm32f3xx/Include/stm32f303x8.h
stm32f3xx/Include/stm32f303xc.h
stm32f3xx/Include/stm32f303xe.h
stm32f3xx/Include/stm32f318xx.h
stm32f3xx/Include/stm32f328xx.h
stm32f3xx/Include/stm32f334x8.h
stm32f3xx/Include/stm32f358xx.h
stm32f3xx/Include/stm32f373xc.h
stm32f3xx/Include/stm32f378xx.h
stm32f3xx/Include/stm32f398xx.h
```

Reply:
It's all the exact same entry:
```
#define RCC_CFGR_PLLXTPRE_Pos                    (17U)
#define RCC_CFGR_PLLXTPRE_Msk                    (0x1UL << RCC_CFGR_PLLXTPRE_Pos) /*!< 0x00020000 */
#define RCC_CFGR_PLLXTPRE                        RCC_CFGR_PLLXTPRE_Msk         /*!< HSE divider for PLL entry */
#define RCC_CFGR_PLLXTPRE_HSE_PREDIV_DIV1        (0x00000000U)                 /*!< HSE/PREDIV clock not divided for PLL entry */
#define RCC_CFGR_PLLXTPRE_HSE_PREDIV_DIV2        (0x00020000U)                 /*!< HSE/PREDIV clock divided by 2 for PLL entry */
```

Reply:
Oh now I understand, this is an alias bit, and this is only relevant for the F1 devices that do *not* have the /1-/16 divider.